docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -98,6 +98,11 @@ children:
     - title: 'Run and Test it'
       url: soap-calculator-tutorial-run-and-test.html
       output: 'web, pdf'
+
+  - title: 'Exposing GraphQL APIs'
```

Comment:
Is this still a Tutorial? I feel that "Exposing GraphQL APIs" and "Deploying to IBM Cloud" don't belong to Tutorials section, I would certainly not look for these pages in Tutorials myself!

Can we introduce a new section "Guides" please and move both pages there?

Reply:
How about turning them into a **How to** sections ?

Reply:
`How to` sounds good to me.